src/libopensc/pkcs15-pin.c

```diff
@@ -306,6 +306,8 @@ sc_pkcs15_verify_pin(struct sc_pkcs15_card *p15card, struct sc_pkcs15_object *pi
 	 * in this case, to avoid error or unnecessary pin prompting on pinpad,
 	 * check if the PIN has been already verified and the access condition
 	 * is still open on card.
+	 *
+	 * But if context specific login we need to use pin pad reader
```

Comment:
this comment can be removed

Reply:
I left it in there on purpose. I can remove it.

Reply:
Comment has been removed.